js/_initial_state.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+
+	activeStep: 0,
+
+	environment: "production",
+	deployment_type: "Code only",
+	deployment_estimate: "2 min",
+
+	summaryOfChanges: "",
+
+	deployment: {
+		started: false
+	},
+
+	git: {
+		selectedType: "",
+		selectedRef: "",
+		isFetching: false,
+		lastUpdated: 0,
+		list: []
+	},
+
+	approval: {
+
+		bypassed: false,
+		bypassed_time: "",
+
+		rejected: false,
```

Comment:
I'm wondering if it's worth refactoring the flags to factor out possibilities like "rejected: true", "approved: true", and "deployment: true" being set at the same time. Maybe we should clarify the state machine and have it the same in the both frontend and backend?